lib/plugins/aws/invokeLocal/index.js

```diff
@@ -178,8 +178,9 @@ class AwsInvokeLocal {
     }
 
     if (runtime.startsWith('nodejs')) {
-      const handlerPath = handler.split('.')[0];
-      const handlerName = handler.split('.')[1];
+      const handlerComponents = handler.split(/\./);
+      const handlerPath = handlerComponents.slice(0, -1).join('.');
+      const handlerName = handlerComponents.pop();
```

Comment:
Probably less convoluted will be that way (?)

```suggestion
      const handlerSeparatorIndex = handler.lastIndexOf('.');
      const handlerPath = handler.slice(0, handlerSeparatorIndex);
      const handlerName = handler.slice(handlerSeparatorIndex + 1);
```

Reply:
Hi @medikoo , thank you very much for your feedback, it's appreciated! Your suggestion does look better so I'll adapt the code.